### PR TITLE
CLI: Update help string

### PR DESCRIPTION
**Pull request by Aumkar**
Added examples of most used commands with options. It will displayed when the user asks `--help` on the `pcc` as well on the sub-command.
Ex.
```
➜  cli git:(od/cli/update-help-string) pcc --help
pcc <cmd>

Commands:
  pcc init <project_directory> [options]  Sets up project with required files.
  pcc token <cmd> [options]               Manage tokens for a PCC project.
  pcc site <cmd> [options]                Manage sites for a PCC project.
  pcc document <cmd> [options]            Manage documents for a PCC Project.
  pcc whoami                              Print information about yourself.
  pcc login                               Logs you in you to PCC client.
  pcc logout                              Logs you out you from PCC client.

Options:
  --version  Show version number  [boolean]
  --help     Show help  [boolean]

Examples:

  Create project with nextjs template
  $ pcc init new_proj

  Create project with gatsby template
  $ pcc init new_proj --template=gatsby

  Create project using pnpm package manager
  $ pcc init new_proj --use-pnpm

  Create Typescript project and setup ESLint in it
  $ pcc init new_proj --ts --eslint

  Create project without installing dependencies
  $ pcc init new_proj --noInstall

  Create project and provide site ID to pre-populate .env file with
  $ pcc init new_proj --site-id 8YUsfuf1EhjLGhswM49q

  Create new token
  $ pcc token create

  Create new site
  $ pcc site create --url test-site.com

  Get webhooks event delivery logs for a site
  $ pcc site webhooks history 8YUsfuf1EhjLGhswM49q

  Generate preview link for given document ID
  $ pcc document preview 8MwijBYyp3B41slkdfjalkdfdziXkjyynTREdst8FauQ

  Get details of logged-in user
  $ pcc whoami

  Login the user
  $ pcc login

  Logout the user
  $ pcc logout
➜  cli git:(od/cli/update-help-string) pcc init --help
pcc init <project_directory> [options]

Sets up project with required files.

Positionals:
  <project_directory>  The project directory in which setup should be done.  [string]

Options:
  --version          Show version number  [boolean]
  --help             Show help  [boolean]
  --template         Template from which files should be copied.  [string] [required] [choices: "nextjs", "gatsby", "vue"] [default: "nextjs"]
  --appName          "package.json" compatible name for the project.  [string]
  --noInstall        Do not install any dependencies.  [boolean] [default: false]
  --use-npm          Use NPM package manager for installing dependencies.  [boolean] [default: true]
  --use-pnpm         Use PNPM package manager for installing dependencies.  [boolean] [default: false]
  --use-yarn         Use Yarn package manager for installing dependencies.  [boolean] [default: false]
  --silent           Skips all console output except for errors and return value from actions.  [boolean] [default: false]
  --non-interactive  Skips asking any questions to the user.  [boolean] [default: false]
  --site-id          Id of site to pre-populate .env file with.  [string] [default: false]
  --eslint           Initialize with eslint config.  [boolean] [default: false]
  --ts               Initialize as a Typescript project.  [boolean] [default: false]
  --verbose          Print verbose logs.  [boolean] [default: true]

Examples:

  Create project with nextjs template
  $ pcc init new_proj

  Create project with gatsby template
  $ pcc init new_proj --template=gatsby

  Create project using pnpm package manager
  $ pcc init new_proj --use-pnpm

  Create Typescript project and setup ESLint in it
  $ pcc init new_proj --ts --eslint

  Create project without installing dependencies
  $ pcc init new_proj --noInstall

  Create project and provide site ID to pre-populate .env file with
  $ pcc init new_proj --site-id 8YUsfuf1EhjLGhswM49q
➜  cli git:(od/cli/update-help-string)
➜  cli git:(od/cli/update-help-string)

```

### CODE REVIEW

File: packages/cli/src/cli/commands/documents.ts [hunk 1]
```diff
@@ -114,3 +114,9 @@ export const generatePreviewLink = errorHandler<GeneratePreviewParam>(
     logger.log(chalk.green(buildLink));
   },
 );
+export const DOCUMENT_EXAMPLES = [
+  {
+    description: "Generate preview link for given document ID",
+    command: "$0 document preview 8MwijBYyp3B41slkdfjalkdfdziXkjyynTREdst8FauQ",
```
Comment (kevinstubbs) [inline packages/cli/src/cli/commands/documents.ts:120]: Use a fake id.

File: packages/cli/src/cli/commands/init.ts [hunk 2]
```diff
@@ -255,3 +255,31 @@ export default errorHandler<{
...
+    command: "pcc init new_proj --ts --eslint",
+  },
+  {
+    description: "Create project without installing dependencies",
+    command: "pcc init new_proj --noInstall",
+  },
+  {
+    description:
+      "Create project and provide site ID to pre-populate .env file with",
+    command: "pcc init new_proj --site-id 8YUsfuf1EhjLGhswM49q",
```
Comment (kevinstubbs) [inline packages/cli/src/cli/commands/init.ts:283]: Fake id.

File: packages/cli/src/cli/commands/sites.ts [hunk 3]
```diff
@@ -125,3 +125,14 @@ export const configurableSiteProperties = [
...
 ] as const;
+
+export const SITE_EXAMPLES = [
+  {
+    description: "Create new site",
+    command: "$0 site create --url test-site.com",
+  },
+  {
+    description: "Get webhooks event delivery logs for a site",
+    command: "$0 site webhooks history 8YUsfuf1EhjLGhswM49q",
```
Comment (kevinstubbs) [inline packages/cli/src/cli/commands/sites.ts:136]: Fake id.

File: packages/cli/src/cli/commands/documents.ts (hunk 1 above)
Comment (Aumkar) [reply packages/cli/src/cli/commands/documents.ts:120]: I copied the original one and made changes to ensure it doesn't exist.
Anyways, I will update it so it explicitly appears like a fake id.

**Review by kevinstubbs — APPROVED** [review]

**Pull request merged**